lib/game.rb

```diff
@@ -0,0 +1,96 @@
+require_relative 'board'
+class Game
+  attr_reader :players, :board
+  attr_accessor :active_player
+
+  def initialize
+    @board = Board.new
+    @players = { 'Player 1' => 'X', 'Player 2' => 'O' }
+    @active_player = 'Player 1'
+  end
+
+  def add(column)
+    row = 5
+    while row >= 0
+      if @board.positions[row][column].nil?
+        @board.positions[row][column] = @players[@active_player]
+        return
+      end
+      row -= 1
+    end
+  end
+
+  def change_player
+    @active_player = if @active_player == 'Player 1'
+                       'Player 2'
+                     else
+                       'Player 1'
+                     end
+  end
+
+  def column_full?(column)
+    return false if @board.positions[0][column].nil?
```

Comment:
1. This looks like it should be`Board#column_full?(column_index)`
2. `@board.positions[0][column].nil?` returns a boolean, and as such `return true / return false` is overly verbose. Whole method could have been
```
def column_full?(column)
  return @board.positions[0][column].present?
end
```